specification/maps/data-plane/Microsoft.Maps/Weather/preview/1.0/weather.json

```diff
@@ -1607,6 +1970,235 @@
         }
       }
     },
+    "DailyClimoActuals": {
+      "type": "object",
+      "properties": {
+        "date": {
+          "type": "string",
+          "description": "Date and time of the current observation displayed in ISO 8601 format, for example, 2019-10-27T19:39:57-08:00."
```

Comment:
`"format": "date-time"` will be useful so then the codegen can generate date types like: https://github.com/ambientlight/azure-sdk-for-go/blob/825aa43364c936dfde70094d465ab8e7e07eef10/services/preview/maps/1.0/weather/models.go#L397

Reply:
`"format": "date-time"` is added.